src/xunit.performance.metrics/BenchmarkDurationEvaluator.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
It looks like you're just splitting these files up.  I'd be interested to understand why, but I don't have a problem with this.